docs/api/proposed_endpoints.md

```diff
@@ -226,3 +226,150 @@ Status: 200 OK
     "posted_time": null
 }
 ```
+
+
+# Registration Endpoints
```

Comment:
Segregate the below endpoints for `# User Auth` and `# Current user actions`. See `endpoints.md` on how it is done.